.github/workflows/ci-pipeline.yml

```
@@ -145,6 +145,12 @@ jobs:
               echo "error=Please ensure all *.tag params are skipped (@skip) in the bitnami/${CHART}/values.yaml file"
               exit "$exit_code"
             fi
+            # Ensuring no unprintable characters are present in values.yaml
+            ! grep -q "[^[:print:]]" "bitnami/${CHART}/values.yaml" || exit_code=$?
```

```suggestion
            ! grep --quiet "[^[:print:]]" "bitnami/${CHART}/values.yaml" || exit_code=$?
```

The rest of `grep`s in the file use the long flag variant